### DISCUSSION TITLE
How to control setupCommands in launch.json based on target?

### DISCUSSION THREAD

**Discussion by jonnysoe** [microsoft/vscode-cpptools#12549] [Q&A]
My project have multiple targets, and a few of them will fork at some point in time, how do I configure CMake Tools to enable `"set follow-fork-mode child"` when `${command:cmake.buildTargetName}` equals to the intended target I want to follow child process instead of parent process?

**Comment by gcampbell-msft** [comment]
I believe this question may be better answered by @sean-mcmanus and the cpp tools team regarding the debugger settings. We don't directly control that.

I'll transfer to the cpptools extension. @sean-mcmanus Any ideas?
